deploy/olm/csv-rhel/storageos.clusterserviceversion.yaml

```diff
@@ -82,89 +82,55 @@ metadata:
-    reserved CPU core and 2GB of free memory. There are minimal external
-    dependencies, and no custom kernel modules.
-
-    After StorageOS is installed, please register for a free Developer
-    license to enable 5TiB of capacity and HA with synchronous
-    replication by following the instructions
-    [here](https://docs.storageos.com/docs/operations/licensing). For
-    additional capacity, features and support plans contact
-    sales@storageos.com.
-
-    ## Highlighted Features
-
-    * **High Availability** - synchronous replication insulates you from node failure.
-
-    * **Delta Sync** - replicas out of sync due to transient failures only transfer changed blocks.
-
-    * **Multiple AccessModes** - dynamically provision ReadWriteOnce or ReadWriteMany volumes.
-
-    * **Rapid Failover** - quickly detects node failure and automates recovery actions without administrator intervention.
-    
-    * **Data Encryption** - both in transit and at rest.  
-
-    * **Scalability** - disaggregated consensus means no single scheduling point of failure.
-
-    * **Thin provisioning** - Only consume the space you need in a storage pool.
-
-    * **Data reduction** - transparent inline data compression to reduce the amount of storage used in a backing store as well as reducing the network bandwidth requirements for replication.
-
-    * **Flexible configuration** - all features can be enabled per volume, using PVC and StorageClass labels.
-
-    * **Multi-tenancy** - fully supports standard Namespace and RBAC methods.
-
-    * **Observability & instrumentation** - log streams for observability and Prometheus support for instrumentation.
-
-    * **Deployment flexibility** - scale up or scale out storage based on application requirements. Works with any infrastructure – on-premises, VM, bare metal or cloud.
-
-    ## Prerequisites
-
-    [StorageOS Prerequisites Docs](https://docs.storageos.com/docs/prerequisites)
-
-    ## Required Parameters
-
-    * `secretRefName` - the name of a secret that contains keys for the
-    credentials
-    ([documentation](https://docs.storageos.com/docs/reference/cluster-operator/examples))
-    * `secretRefNamespace` - the namespace where the api credentials secret is
-    stored
-    * `kvBackend.address` - address of the etcd cluster
-    ([documentation](https://docs.storageos.com/docs/prerequisites/etcd/))
-
-    ## About StorageOS
-
-    StorageOS is a software-defined cloud native storage platform
-    delivering persistent storage for Kubernetes. StorageOS is built from
-    the ground-up with no legacy restrictions to give enterprises working
-    with cloud native workloads a scalable storage platform with no
-    compromise on performance, availability or security. For additional
-    information, visit [www.storageos.com](www.storageos.com).
+  description: "StorageOS is a cloud native, software-defined storage platform that\ntransforms
```

Comment:
I more like the previous version of this description. Should we use `|` instead of `"`?

Reply:
This seems to be coming from some updated tool, not sure which one, probably yq, although we use a pinned version. I tried previewing this CSV in the https://operatorhub.io/preview and the result looks same. We can manually change it back as this may be the last time we are touching this or keep it as it is.
Again, the new tooling in the new operator using kustomize makes all these better.

Reply:
Ok i see, i have approved the change, it is up to @croomes to change it if he agrees.

Reply:
I'm not sure why, but the generation tools changed this.

Reply:
I've reverted - I prefer the old way too.